apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/mapper/OrganizationMapperTest.java

```diff
@@ -0,0 +1,111 @@
+package org.sagebionetworks.openchallenges.organization.service.model.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.OrganizationDto;
+import org.sagebionetworks.openchallenges.organization.service.model.entity.OrganizationEntity;
+
+public class OrganizationMapperTest {
+  @Test
+  public void ConvertToEntity_ShouldReturnDtoProperties_WhenDtoPropertiesPassed() {
+
+    // create new Dto with properties to be copied to organization entity
+    OrganizationDto dto = new OrganizationDto();
+    dto.setName("Test Organization");
+    dto.setDescription("This is a test organization");
+    dto.setId(4599L);
+    dto.setEmail("testemail@test.edu");
+    dto.setLogin("login");
+    dto.setAvatarKey("avatar Key");
+    dto.setWebsiteUrl("url");
+    dto.setChallengeCount(7);
+    dto.setAcronym(null);
+
+    OrganizationMapper mapper = new OrganizationMapper();
+
+    OrganizationEntity entity = mapper.convertToEntity(dto);
+
+    Long actualId = entity.getId();
+    String actualName = entity.getName();
+    String actualEmail = entity.getEmail();
+    String actualLogin = entity.getLogin();
+    String actualAvatarKey = entity.getAvatarKey();
+    String actualWebsiteUrl = entity.getWebsiteUrl();
+    Integer actualChallengeCount = entity.getChallengeCount();
+    String actualDescription = entity.getDescription();
+    String actualAcronym = entity.getAcronym();
+
+    Long actualId2 = dto.getId();
+    String actualName2 = dto.getName();
+    String actualEmail2 = dto.getEmail();
+    String actualLogin2 = dto.getLogin();
+    String actualAvatarKey2 = dto.getAvatarKey();
+    String actualWebsiteUrl2 = dto.getWebsiteUrl();
+    Integer actualChallengeCount2 = dto.getChallengeCount();
+    String actualDescription2 = dto.getDescription();
+    String actualAcronym2 = dto.getAcronym();
+
+    // verify the entity properties were copied
+    Assertions.assertEquals(actualId, actualId2);
+    Assertions.assertEquals(actualName, actualName2);
+    Assertions.assertEquals(actualEmail, actualEmail2);
+    Assertions.assertEquals(actualLogin, actualLogin2);
+    Assertions.assertEquals(actualAvatarKey, actualAvatarKey2);
+    Assertions.assertEquals(actualWebsiteUrl, actualWebsiteUrl2);
```

Comment:
The code would be more compact and readable if you do:

```
Assertions.assertEquals(dto,getId(), entity.getId());
...
```

Reply:
Updated the code to reflect the requested changes.